api/controllers/SocketController.js

```diff
@@ -15,8 +17,15 @@ function test(message) {
 	this.socket.emit('action', { type: 'SOCKET_TEST', message: 'Hello from server!' });
 }
 
-function addTicket(ticket) {
-	this.socket.emit('action', { type: 'SOCKET_NEW_TICKET', ticket });
+async function addTicket(ticket) {
+	const { requestorId, description, tableNum, contactInfo } = ticket;
+	try{
+		const newTicket = await Ticket.create(requestorId, description, tableNum, contactInfo);
+		this.socket.emit('action', { type: 'SOCKET_NEW_TICKET', newTicket: newTicket });
```

Comment:
Less redundant: `this.socket.emit('action', { type: 'SOCKET_NEW_TICKET', newTicket });`